EnglishFilter/sections/antiadblock.txt

```diff
@@ -596,7 +596,7 @@ uaclips.com,kzclip.com,ru-clip.com,ruclip.com###morev
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/90092
 @@||anonigviewer.com/assets/js/advertisements.js
-anonigviewer.com#%#//scriptlet("set-constant", "adblock", "false")
+anonigviewer.com#%#//scriptlet("set-constant", "adblock", "false"
```

Comment:
Oh, there is missed `)` at the end.

Reply:
Restored. https://github.com/AdguardTeam/AdguardFilters/pull/90429/commits/5f76190a52a569afec363b34a27adfa8767580fa